rasa/nlu/training_data/util.py

```diff
@@ -49,3 +51,45 @@ def get_file_format(resource_name: Text) -> Text:
         return fformat
 
     return "json"
+
+
+def remove_untrainable_entities_from(example: Dict[Text, Any]) -> None:
+    """Remove untrainable entities from serialised training example `example`.
+
+    Entities with an untrainable extractor will be removed. Untrainable extractors
+    are `DucklingHTTPExtractor`, `MitieEntityExtractor` and `SpacyEntityExtractor`.
+
+    Args:
+        example: Serialised training example to inspect.
+    """
+
+    from rasa.nlu.extractors.duckling_http_extractor import DucklingHTTPExtractor
+    from rasa.nlu.extractors.mitie_entity_extractor import MitieEntityExtractor
+    from rasa.nlu.extractors.spacy_entity_extractor import SpacyEntityExtractor
+
+    untrainable_entity_extractors = [
+        DucklingHTTPExtractor.__name__,
+        SpacyEntityExtractor.__name__,
+        MitieEntityExtractor.__name__,
+    ]
+
+    example_entities = example.get(ENTITIES_ATTRIBUTE)
+
+    if not example_entities:
+        # example contains no entities, so there's nothing to do
+        return None
+
+    trainable_entities = []
+
+    for entity in example_entities:
+        if entity.get(EXTRACTOR_ATTRIBUTE) in untrainable_entity_extractors:
+            logger.debug(
+                f"Excluding entity '{json.dumps(entity)}' from training data. "
+                f"Entity examples extracted by the following classes are not "
+                f"dumped to training data in markdown format: "
+                f"`{'`, `'.join(untrainable_entity_extractors)}`."
+            )
```

Comment:
hmm, not really since both functions that call it do so on the level of individual examples. passing a whole list of examples into this function would mean looping over all training examples twice which is unnecessary. I think this is fine, especially since it's a debug message. wdyt?

Reply:
No problem, if it were easy, then id implement it, but yeah definitely not worth changing the functionality for 👍